camelot/admin/action/list_action.py

```diff
@@ -330,10 +337,12 @@ class DuplicateSelection( EditAction ):
     tooltip = _('Duplicate')
     verbose_name = _('Duplicate')
     name = 'duplicate_selection'
-    
+
     def model_run( self, model_context ):
         from camelot.view import action_steps
         admin = model_context.admin
+        if not admin.entity.is_editable():
```

Comment:
steek het checken van die editability en het raisen vd exception aub in een method op EditAction, zodat die overal kan gecalled w.

Reply:
done